### DISCUSSION TITLE
sidebar 还会支持 collapsable 功能吗？

### DISCUSSION THREAD

**Discussion by jevantang** [vuepress/core#396] [Q&A] [answered]
我的产品也是开源软件，文档涉及到几十个数据表和 API 接口，如此多的数量，sidebar 非常的长，所以我对数据表和 API 文档进行了归类，这样 sidebar 非常的清爽了，不至于太多列表，影响了观感。

但是升级到 2.0 版本之后，不能 collapsable 了，长长的侧边栏，浏览体验不是很友好。所以 v2 还会支持 collapsable 吗？

API 例子： [https://fresns.cn/api/](https://fresns.cn/api/)
数据字典例子： [https://fresns.cn/database/](https://fresns.cn/database/)

侧边栏很长。

**Comment by meteorlxy** [comment]
可以支持，只是一直没有做。（而且其实一直没人正经提过 Feature Request

**Reply by jevantang** [reply]
好的，那我就去提一个

**Comment by Mister-Hope** [comment, marked as answer]
Already supported
